resources/views/user/rsi/comm_links/edit.blade.php

```diff
@@ -1,6 +1,6 @@
 @extends('user.layouts.default_wide')
 
-@section('title', __('Comm-Link').' - '.$commLink->title.' bearbeiten')
+@section('title', __('Comm-Link').' - '.$commLink->title. ' ' . __('(bearbeiten)'))
```

Comment:
Are you sure that `(bearbeiten)` is the correct translation key?  
Or do you mean `__('bearbeiten')`?